Makefile

```diff
@@ -16,21 +16,28 @@ PYTHON ?= $(shell which python3)
 MAKEFILE_DIR := $(realpath $(dir $(lastword $(MAKEFILE_LIST))))
 PY3_VER ?= $(shell $(PYTHON) -c "import sys;print('%d%d' % sys.version_info[:2])")
 OS := $(shell uname -s)
+ARCH := $(shell uname -p)
 
 ifeq ($(PYTHON),)
 $(error PYTHON must be set)
 endif
 
-# Allowed CPU values: k8, armv7a, aarch64, darwin
 ifeq ($(OS),Linux)
 CPU ?= k8
 else ifeq ($(OS),Darwin)
+ifeq ($(ARCH),arm)
+CPU ?= darwin_arm64
+DARWIN_CPU := darwin_arm64
+else
 CPU ?= darwin
```

Comment:
Thanks @yasdum!
I'm not super familiar with MacOS, why is there a need for a `darwin` and a `darwin_x86_64`?